src/plugins/plotting/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+ign_gui_add_plugin(GuiPlotting
```

Comment:
How about calling this plugin `TransportPlotting`? That may be clearer to end users.